app/views/users/_form.html.erb

```diff
@@ -13,26 +13,50 @@
   <br>
 
   <div class="field">
-    <%= f.label :first_name %>
-    <%= f.text_field :first_name %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.label :first_name %>
+      </div>
+      <%= f.text_field :first_name %>
+    </div>
   </div>
   <div class="field">
-    <%= f.label :last_name %>
-    <%= f.text_field :last_name %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.label :last_name %>
+      </div>
+      <%= f.text_field :last_name %>
+    </div>
   </div>
   <div class="field">
-    <%= f.label :email %>
-    <%= f.text_field :email %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.label :email %>
+      </div>
+      <%= f.text_field :email %>
+    </div>
   </div>
   <div class="field">
-    <%= f.label :password %>
-    <%= f.password_field :password %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.label :password %>
+      </div>
+      <%= f.password_field :password %>
+    </div>
   </div>
   <div class="field">
-   <%= f.label :password_confirmation %>
-   <%= f.password_field :password_confirmation %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.label :password_confirmation %>
+      </div>
+      <%= f.password_field :password_confirmation %>
+    </div>
   </div>
   <div class="actions">
-    <%= f.submit class: "btn btn-success" %>
+    <div class="form-group">
+      <div class="col-sm-3">
+        <%= f.submit class: "btn btn-success" %>
```

Comment:
looks good. we can talk about how to display the create and back button. right now they aren't side by side but create isn't inline with the back button.